api/src/main/java/com/okta/sdk/client/ClientBuilder.java

```diff
@@ -206,6 +208,11 @@
     String DEFAULT_CLIENT_PROXY_HOST_PROPERTY_NAME = "okta.client.proxy.host";
     String DEFAULT_CLIENT_PROXY_USERNAME_PROPERTY_NAME = "okta.client.proxy.username";
     String DEFAULT_CLIENT_PROXY_PASSWORD_PROPERTY_NAME = "okta.client.proxy.password";
+    String DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME = "okta.client.authorizationMode";
+    String DEFAULT_CLIENT_CLIENT_ID_PROPERTY_NAME = "okta.client.clientId";
+    String DEFAULT_CLIENT_SCOPES_PROPERTY_NAME = "okta.client.scopes";
+    String DEFAULT_CLIENT_PRIVATE_KEY_KEY_FILE_PATH_PROPERTY_NAME = "okta.client.privateKey.keyFilePath";
+    String DEFAULT_CLIENT_PRIVATE_KEY_ALGORITHM_PROPERTY_NAME = "okta.client.privateKey.algorithm";
```

Comment:
Again, do we really need `algorithm`?

Reply:
I mentioned this too, we should be able to detect the alg automatically from the cert and remove the need for this prop

Reply:
Noted guys, will get rid of this and refactor code around this.